### PR TITLE
use BigNumber and update withdraw method

### PR DESCRIPTION
**Pull request by andrewdamelio**
@justinbarry see https://github.com/AugurProject/masterchef-gnosis-safe-rewards-ui/compare/UI-updates?expand=1#diff-26ad4b834941d9b19ebf9db8082bd202aaf72ea0ddea85f5a8a0cb3c729cc6f2L121-L122

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -3,12 +3,15 @@ import styled from 'styled-components'
...
 import MasterChefABI from './MasterChef.json';
+import WMaticABI from './WMatic.json';
 // @ts-ignore
 import Web3 from 'web3';
+import { BigNumber } from "ethers";
 
 const POLYGON_NETWORK_RPC = "https://polygon-rpc.com"
 const ZERO_ADDRESS = "0x0000000000000000000000000000000000000000"
-
+const WMATIC_TOKEN_ADDRESS = "0x0d500b1d8e8ef31e21c99d1db9a6444d3adf1270";
```
Comment (justinbarry) [inline src/App.tsx:13]: Can we make these configurable somehow? Or at least overridable for testing?

File: src/App.tsx [hunk 2]
```diff
@@ -116,16 +119,19 @@ const SafeApp = (): React.ReactElement => {
     } catch (e) {
       console.error(e)
     }
-  }, [contract.methods, rewardsContract, withdrawAmount, sdk])
+  }, [contract.methods, rewardsContract, sdk])
 
   const addRewardsTx = useCallback(async () => {
+    const rewardsPerPeriod = BigNumber.from(10).pow(18).mul(rewardsPerMarket);
+    const rewardPeriods = BigNumber.from(rewardDaysPerMarket);
+    const rewardEarlyDepositBonus= BigNumber.from(10).pow(18).mul(earlyDepositBonusRewards);
```
Comment (justinbarry) [inline src/App.tsx:127]: LOVE THIS. Can we a note in the UI that this is happening?

**Review by justinbarry — APPROVED** [review]
Couple thoughts. Otherwise this looks really good!

File: src/App.tsx (hunk 1 above)
Comment (andrewdamelio) [reply src/App.tsx:13]: will do in another PR, ill make these all switchable via UI

File: src/App.tsx (hunk 2 above)
Comment (andrewdamelio) [reply src/App.tsx:127]: 👍

**Pull request merged**